tests/adapters/switches/juniper_test.py

```diff
@@ -4312,11 +4312,11 @@ def test_get_bond(self):
 
         assert_that(if3.number, equal_to(3))
         assert_that(if3.link_speed, equal_to('1g'))
-        assert_that(if3.interface.name, equal_to("ae3"))
-        assert_that(if3.interface.port_mode, equal_to(TRUNK))
-        assert_that(if3.interface.access_vlan, equal_to(None))
-        assert_that(if3.interface.trunk_native_vlan, equal_to(2000))
-        assert_that(if3.interface.trunk_vlans, equal_to([999, 1000, 1001]))
+        assert_that(if3.name, equal_to('ae3'))
```

Comment:
Further more, this test should have to be checking for backward compaibility maybe, another should be written and just checking that .interface.name is correctly set... this way however you implement the "_hidden_field" is not enforced by the test